config/settings_schema.json

```
@@ -415,35 +397,17 @@
         "type": "range",
         "id": "buttons_shadow_blur_radius",
         "min": 0,
-        "max": 20,
-        "step": 2,
+        "max": 40,
```

```suggestion
        "max": 20,
```
For the same reasons of Inputs